e2e/playwright/share-link.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { expect, test } from '@e2e/playwright/zoo-test'
+
+const isWindows =
+  navigator.platform === 'Windows' || navigator.platform === 'Win32'
+test.describe('Share link tests', () => {
```

Comment:
We may want to take this `@web` or that can happen later.

Reply:
You know what, follow up at https://github.com/KittyCAD/modeling-app/pull/6784